demo/src/app/components/shared/api-docs/api-docs-badge.component.ts

```diff
@@ -11,14 +11,21 @@ const BADGES = {
 
 @Component({
   selector: 'ngbd-api-docs-badge',
-  template: `<h5><span class="badge" [ngClass]="badgeClass">{{text}}</span></h5>`,
+  template: `
+    <h5>
+      <span *ngIf="deprecated" class="badge badge-secondary" >Deprecated {{ deprecated.version }}</span> 
```

Comment:
Shouldn't this be "Deprecated from" or "Deprecated in"?

nit: extra space before `<span>` tag close